sdk/storage/storage-queue/samples/typescript/basic.ts

```diff
@@ -18,15 +18,28 @@ async function main() {
   // SharedKeyCredential is only avaiable in Node.js runtime, not in browsers
   const sharedKeyCredential = new SharedKeyCredential(account, accountKey);
 
+  // DefaultAzureCredential will first look for Azure Active Directory (AAD)
```

Comment:
Is it worth mentioning "Node.js only" everywhere that `DefaultAzureCredential` is used in the samples?

Reply:
I have the comment saying  "Node.js only" on Line 31.

Reply:
It gets lost in the block of text I wrote, maybe it's better if it's at the top of the block in all caps like the other instance that I commented on?

Reply:
Updated.